<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoutineWindow</class>
 <widget class="QDialog" name="RoutineWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoutineWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-image: url(:/image/book.jpg);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/book.jpg);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>70</y>
       <width>591</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>29</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: black;
   }</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:21pt; font-weight:700;&quot;&gt;ROUTINE WINDOW&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>491</width>
       <height>227</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">QLabel{
color: black;
   

       font-size:20px;  
       padding: 1px; 
       font-weight: bold;  
     text-align: center; 
   
}
</string>
        </property>
        <property name="text">
         <string>ADD ROUTINE:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">QLabel{
color: black;
   

       font-size:20px;  
       padding: 1px; 
       font-weight: bold;  
     text-align: center; 
   
}
</string>
        </property>
        <property name="text">
         <string>VIEW ROUTINE:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>VIEW</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="styleSheet">
         <string notr="true">QLabel{
color: black;
   

       font-size:20px;  
       padding: 1px; 
       font-weight: bold;  
     text-align: center; 
   
}
</string>
        </property>
        <property name="text">
         <string>EDIT ROUTINE:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">QLabel{
color: black;
   

       font-size:20px;  
       padding: 1px; 
       font-weight: bold;  
     text-align: center; 
   
}
</string>
        </property>
        <property name="text">
         <string>DELETE ROUTINE:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>DELETE</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>EDIT</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>pushButton</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>label_9</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_10</zorder>
     <zorder>pushButton_4</zorder>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>400</y>
       <width>195</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="home">
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>LOGOUT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back">
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>BACK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;ROUTINE PAGE&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>611</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>631</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">QLabel{
color: black;
   

       font-size:20px;  
       padding: 1px; 
       font-weight: bold;  
     text-align: center; 
   
}
</string>
        </property>
        <property name="text">
         <string>CHOOSE DAY   :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_3">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox QAbstractItemView {
    color: black; /* Color of items in dropdown */
}</string>
        </property>
        <item>
         <property name="text">
          <string>Sunday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tuesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wednesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thursday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Friday</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>-60</x>
       <y>500</y>
       <width>691</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>BACK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>LOGOUT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_7">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>SUMBIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
